### ISSUE TITLE
Some kind of error with bombs?

### ISSUE THREAD

**Issue by jfrogale** [coolAlias/ZeldaSwordSkills#83]
I'm not entirely sure what happened. I was just dicking around in my world and I crash because I picked up an item from the Witchery mod (Wolfsbane seeds, to be precise) I get rid of them in MCEdit, relaunch the game, and get a new error for this mod. I can't seem to open the crash report, so the tiny error "report" is all I can provide. Note, the mod had been working fine for months before, so I'm not sure what is meant by the first line : 
It looks like you are using an unsupported modified version of the game.
The game crashed whilst ticking entity
Error: java.lang.NoSuchMethodError: zeldaswordskills.entity.projectile.EntityBomb.setTime(I)Lzeldaswordskills/entity/projectile/EntityBomb;

Beforehand I had noticed that secret rooms had been opening on their own somehow, as I had gotten the "blast open 50 rooms" achievement after much less than that, maybe 10-20. Maybe this is the reason? Thanks in advance

**Comment by jfrogale** [comment]
Okay, so I found the crash log, but it may be worth noting I know jack about what it all means. If someone does see this, and can help, could you possibly dumb it down to where I can solve the problem? Thanks a million in advance.

---- Minecraft Crash Report ----
// Sorry :(

Time: 11/18/17 6:52 PM
Description: Ticking entity

java.lang.NoSuchMethodError: zeldaswordskills.entity.projectile.EntityBomb.setTime(I)Lzeldaswordskills/entity/projectile/EntityBomb;
	at addon.zeldaswordskills.entity.EntityLandOctorok.func_82196_d(EntityLandOctorok.java:193)
	at net.minecraft.entity.ai.EntityAIArrowAttack.func_75246_d(SourceFile:85)
	at net.minecraft.entity.ai.EntityAITasks.func_75774_a(SourceFile:103)
	at net.minecraft.entity.EntityLiving.func_70619_bc(EntityLiving.java:540)
	at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:1774)
	at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:367)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:1611)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:206)
	at addon.zeldaswordskills.entity.EntityLandOctorok.func_70071_h_(EntityLandOctorok.java:156)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:648)
	at net.minecraft.world.World.func_72870_g(World.java:2034)
	at net.minecraft.world.World.func_72939_s(World.java:1887)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:489)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:636)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at addon.zeldaswordskills.entity.EntityLandOctorok.func_82196_d(EntityLandOctorok.java:193)
	at net.minecraft.entity.ai.EntityAIArrowAttack.func_75246_d(SourceFile:85)
	at net.minecraft.entity.ai.EntityAITasks.func_75774_a(SourceFile:103)
	at net.minecraft.entity.EntityLiving.func_70619_bc(EntityLiving.java:540)
	at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:1774)
	at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:367)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:1611)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:206)
	at addon.zeldaswordskills.entity.EntityLandOctorok.func_70071_h_(EntityLandOctorok.java:156)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:648)
	at net.minecraft.world.World.func_72870_g(World.java:2034)

-- Entity being ticked --
Details:
	Entity Type: zssaddon.landOctorok (addon.zeldaswordskills.entity.EntityLandOctorok)
	Entity ID: 435
	Entity Name: Land Octorok
	Entity's Exact location: -1102.53, 70.00, 2029.74
	Entity's Block location: World: (-1103,70,2029), Chunk: (at 1,4,13 in -69,126; contains blocks -1104,0,2016 to -1089,255,2031), Region: (-3,3; contains chunks -96,96 to -65,127, blocks -1536,0,1536 to -1025,255,2047)
	Entity's Momentum: 0.00, -0.08, 0.00
Stacktrace:
	at net.minecraft.world.World.func_72939_s(World.java:1887)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:489)

-- Affected level --
Details:
	Level name: Survival (mod)
	All players: 1 total; [EntityPlayerMP['jfrogale'/304, l='Survival (mod)', x=-1096.69, y=70.00, z=2035.27]]
	Chunk stats: ServerChunkCache: 737 Drop: 0
	Level seed: -1818276257
	Level generator: ID 00 - default, ver 1. Features enabled: true
	Level generator options: 
	Level spawn location: World: (350,64,156), Chunk: (at 14,4,12 in 21,9; contains blocks 336,0,144 to 351,255,159), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 1611073 game time, 1611073 day time
	Level dimension: 0
	Level storage version: 0x04ABD - Anvil
	Level weather: Rain time: 21807 (now: false), thunder time: 173140 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: true
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:636)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 339765152 bytes (324 MB) / 1073741824 bytes (1024 MB) up to 1073741824 bytes (1024 MB)
	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=16M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 13, tcache: 0, allocated: 15, tallocated: 95
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1566 81 mods loaded, 70 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1566-1.7.10.jar) 
	UCHIJAAAA	Forge{10.13.4.1566} [Minecraft Forge] (forge-1.7.10-10.13.4.1566-1.7.10.jar) 
	UCHIJAAAA	CodeChickenCore{1.0.7.47} [CodeChicken Core] (minecraft.jar) 
	UCHIJAAAA	NotEnoughItems{1.0.5.120} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.5.120-universal (1).jar) 
	UCHIJAAAA	<CoFH ASM>{000} [CoFH ASM] (minecraft.jar) 
	UCHIJAAAA	zeldaswordskills{1.7.10-2.5.1} [Zelda Sword Skills] (1.7.10-zeldaswordskills-2.5.1.jar) 
	UCHIJAAAA	divinerpg{1.4.1.3} [DivineRPG] ([1.7.10][SMP]DivineRPG-1.4.1.3.jar) 
	UCHIJAAAA	bspkrsCore{6.15} [bspkrsCore] ([1.7.10]bspkrsCore-universal-6.15.jar) 
	UCHIJAAAA	FloatingRuins{1.7.10.r02} [FloatingRuins] ([1.7.10]FloatingRuins-universal-1.7.10.r02.jar) 
	UCHIJAAAA	Treecapitator{1.7.10} [Treecapitator] ([1.7.10]Treecapitator-universal-2.0.4.jar) 
	UCHIJAAAA	uncraftingTable{1.4.4-pre10} [Uncrafting Table] ([1.7.10]UncraftingTable-1.4.4-pre10.jar) 
	UCHIJAAAA	BattleTowers{1.5.1} [Battle Towers] (BattleTowers-1.7.10.jar) 
	UCHIJAAAA	bdlib{1.9.4.109} [BD Lib] (bdlib-1.9.4.109-mc1.7.10.jar) 
	UCHIJAAAA	BiomesOPlenty{2.1.0} [Biomes O' Plenty] (BiomesOPlenty-1.7.10-2.1.0.1889-universal (1).jar) 
	UCHIJAAAA	AWWayofTime{v1.3.3} [Blood Magic: Alchemical Wizardry] (BloodMagic-1.7.10-1.3.3-17.jar) 
	UCHIJAAAA	CoFHCore{1.7.10R3.1.4} [CoFH Core] (CoFHCore-[1.7.10]3.1.4-329.jar) 
	UCHIJAAAA	BuildCraft|Core{7.1.21} [BuildCraft] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	BuildCraft|Builders{7.1.21} [BC Builders] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	BuildCraft|Energy{7.1.21} [BC Energy] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	BuildCraft|Factory{7.1.21} [BC Factory] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	BuildCraft|Robotics{7.1.21} [BC Robotics] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	BuildCraft|Silicon{7.1.21} [BC Silicon] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	BuildCraft|Transport{7.1.21} [BC Transport] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	CarpentersBlocks{3.3.8.1} [Carpenter's Blocks] (Carpenter's Blocks v3.3.8.1 - MC 1.7.10.jar) 
	UCHIJAAAA	ChestTransporter{2.0.3} [Chest Transporter] (ChestTransporter-2.0.3-MC1.7.10.jar) 
	UCHIJAAAA	ChickenChunks{1.3.4.16} [ChickenChunks] (ChickenChunks-Mod-1.7.10-Recommended.jar) 
	UCHIJAAAA	DamageIndicatorsMod{3.2.0} [Damage Indicators] (Damage-Indicators-Mod-1.7.10.jar) 
	UCHIJAAAA	dldungeonsjdg{1.9.2} [Doomlike Dungeons] (DoomlikeDungeons-1.9.2-MC1.7.10.jar) 
	UCHIJAAAA	eplus{3.0.2-d} [Enchanting Plus] (EnchantingPlus-1.7.10-3.0.2-d.jar) 
	UCHIJAAAA	ThermalFoundation{1.7.10R1.2.6} [Thermal Foundation] (ThermalFoundation-[1.7.10]1.2.6-118.jar) 
	UCHIJAAAA	ExtraUtilities{1.2.12} [Extra Utilities] (extrautilities-1.2.12.jar) 
	UCHIJAAAA	MetallurgyCore{4.0.5} [Metallurgy Core] (MetallurgyCore-1.7.10-4.0.5.30.jar) 
	UCHIJAAAA	Mantle{1.7.10-0.3.2.jenkins191} [Mantle] (Mantle-1.7.10-0.3.2b.jar) 
	UCHIJAAAA	MineFactoryReloaded{1.7.10R2.8.1} [MineFactory Reloaded] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	ThermalExpansion{1.7.10R4.1.5} [Thermal Expansion] (ThermalExpansion-[1.7.10]4.1.5-248.jar) 
	UCHIJAAAA	Waila{1.5.10} [Waila] (Waila-1.5.10_1.7.10.jar) 
	UCHIJAAAA	TConstruct{1.7.10-1.8.5.build957} [Tinkers' Construct] (TConstruct-1.7.10-1.8.5.jar) 
	UCHIJAAAA	Metallurgy{4.0.9} [Metallurgy 4] (Metallurgy-1.7.10-4.0.9.148.jar) 
	UCHIJAAAA	harvestcraft{1.7.10j} [Pam's HarvestCraft] (Pam's HarvestCraft 1.7.10Lb.jar) 
	UCHIJAAAA	ExtraTiC{1.4.6} [ExtraTiC] (ExtraTiC-1.7.10-1.4.6.jar) 
	UCHIJAAAA	FastCraft{1.21} [FastCraft] (fastcraft-1.21.jar) 
	UCHIJAAAA	fastleafdecay{1.1} [Fast Leaf Decay] (FastLeafDecay-1.7.10-1.1.jar) 
	UCHIJAAAA	iChunUtil{4.1.0} [iChunUtil] (iChunUtil-4.1.0.jar) 
	UCHIJAAAA	InfernalMobs{1.6.0} [Infernal Mobs] (InfernalMobs-1.7.10.jar) 
	UCHIJAAAA	LunatriusCore{1.1.2.21} [LunatriusCore] (LunatriusCore-1.7.10-1.1.2.21-universal.jar) 
	UCHIJAAAA	InGameInfoXML{2.8.1.82} [InGame Info XML] (InGameInfoXML-1.7.10-2.8.1.82-universal.jar) 
	UCHIJAAAA	inventorytweaks{1.59-dev-152-cf6e263} [Inventory Tweaks] (InventoryTweaks-1.59-dev-152.jar) 
	UCHIJAAAA	IronChest{6.0.62.742} [Iron Chest] (ironchest-1.7.10-6.0.62.742-universal.jar) 
	UCHIJAAAA	magicalcrops{4.0.0_PUBLIC_BETA_3} [Magical Crops] (magicalcrops-4.0.0_PUBLIC_BETA_3.jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatBuildCraft{1.7.10R2.8.1} [MFR Compat: BuildCraft] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatForgeMicroblock{1.7.10R2.8.1} [MFR Compat: ForgeMicroblock] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatThermalExpansion{1.7.10R2.8.1} [MFR Compat: Thermal Expansion] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatTConstruct{1.7.10R2.8.1} [MFR Compat: Tinkers' Construct] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	TwilightForest{2.3.7} [The Twilight Forest] (twilightforest-1.7.10-2.3.7 (1).jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatTwilightForest{1.7.10R2.8.1} [MFR Compat: TwilightForest] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatVanilla{1.7.10R2.8.1} [MFR Compat: Vanilla] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	me{1.3.2.1} [More Enchantments] (MoreEnchantments-1.7.10-1.3.2.1.jar) 
	UCHIJAAAA	AencEx{0.0.1} [AencEx] (MoreEnchantments-1.7.10-1.3.2.1.jar) 
	UCHIJAAAA	Morph{0.9.3} [Morph] (Morph-Beta-0.9.3.jar) 
	UCHIJAAAA	NetherOres{1.7.10R2.3.1} [Nether Ores] (NetherOres-[1.7.10]2.3.1-22.jar) 
	UCHIJAAAA	pressure{1.3.0.130} [Pressure Pipes] (pressure-1.3.0.130-mc1.7.10.jar) 
	UCHIJAAAA	projectbench{0.5.0} [Project Bench Updated] (ProjectBench-Update-1.7.10-0.5.jar) 
	UCHIJAAAA	AS_Ruins{15.4} [Ruins Spawning System] (Ruins-1.7.10.jar) 
	UCHIJAAAA	ShetiPhianCore{v3} [ShetiPhian-Core] (ShetiPhianCore-v3-beta2.jar) 
	UCHIJAAAA	TiCTooltips{1.2.3} [TiC Tooltips] (TiCTooltips-mc1.7.10-1.2.3.jar) 
	UCHIJAAAA	WailaHarvestability{1.1.2} [Waila Harvestability] (WailaHarvestability-mc1.7.x-1.1.2.jar) 
	UCHIJAAAA	witchery{0.24.1} [Witchery] (witchery-1.7.10-0.24.1.jar) 
	UCHIJAAAA	XaeroMinimap{1.12.5} [Xaero's Minimap] (Xaeros_Minimap_1.12.5_Forge_1.7.10.jar) 
	UCHIJAAAA	zssaddon{1.7.10-alpha-0.0.5-2} [Zelda Sword Skills - Addon] (zeldaswordskillsaddon-v0.0.5a-2-.jar) 
	UD	MineFactoryReloaded|CompatAppliedEnergistics{1.7.10R2.8.1} [MFR Compat: Applied Energistics] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatAtum{1.7.10R2.8.1} [MFR Compat: Atum] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatBackTools{1.7.10R2.8.1} [MFR Compat: BackTools] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatChococraft{1.7.10R2.8.1} [MFR Compat: Chococraft] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatExtraBiomes{1.7.10R2.8.1} [MFR Compat: ExtraBiomes] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatForestry{1.7.10R2.8.1} [MFR Compat: Forestry] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatIC2{1.7.10R2.8.1} [MFR Compat: IC2] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatProjRed{1.7.10R2.8.1} [MFR Compat ProjectRed] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatRailcraft{1.7.10R2.8.1} [MFR Compat: Railcraft] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatSufficientBiomes{1.7.10R2.8.1} [MFR Compat: Sufficient Biomes] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatThaumcraft{1.7.10R2.8.1} [MFR Compat: Thaumcraft] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	CoFHCore: -[1.7.10]3.1.4-329
	ThermalFoundation: -[1.7.10]1.2.6-118
	Mantle Environment: Environment healthy.
	MineFactoryReloaded: -[1.7.10]2.8.1-174
	ThermalExpansion: -[1.7.10]4.1.5-248
	TConstruct Environment: Environment healthy.
	NetherOres: -[1.7.10]2.3.1-22
	List of loaded APIs: 
		* Baubles|API (1.0.1.10) from ThermalFoundation-[1.7.10]1.2.6-118.jar
		* BiomesOPlentyAPI (1.0.0) from BiomesOPlenty-1.7.10-2.1.0.1889-universal (1).jar
		* BloodMagicAPI (1.3.3-13) from BloodMagic-1.7.10-1.3.3-17.jar
		* BuildCraftAPI|blocks (1.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|blueprints (1.5) from buildcraft-7.1.21.jar
		* BuildCraftAPI|boards (2.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|core (1.0) from extrautilities-1.2.12.jar
		* BuildCraftAPI|crops (1.1) from buildcraft-7.1.21.jar
		* BuildCraftAPI|events (2.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|facades (1.1) from buildcraft-7.1.21.jar
		* BuildCraftAPI|filler (4.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|fuels (2.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|gates (4.1) from buildcraft-7.1.21.jar
		* BuildCraftAPI|items (1.1) from buildcraft-7.1.21.jar
		* BuildCraftAPI|library (2.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|lists (1.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|power (1.3) from buildcraft-7.1.21.jar
		* BuildCraftAPI|recipes (3.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|robotics (3.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|statements (1.1) from buildcraft-7.1.21.jar
		* BuildCraftAPI|tablet (1.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|tiles (1.2) from buildcraft-7.1.21.jar
		* BuildCraftAPI|tools (1.0) from extrautilities-1.2.12.jar
		* BuildCraftAPI|transport (4.1) from buildcraft-7.1.21.jar
		* CarpentersBlocks|API (3.3.7) from Carpenter's Blocks v3.3.8.1 - MC 1.7.10.jar
		* CoFHAPI (1.7.10R1.0.13B1) from extrautilities-1.2.12.jar
		* CoFHAPI|block (1.7.10R1.0.13B1) from extrautilities-1.2.12.jar
		* CoFHAPI|core (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|energy (1.7.10R1.0.2) from buildcraft-7.1.21.jar
		* CoFHAPI|fluid (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|inventory (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|item (1.7.10R1.0.13B1) from extrautilities-1.2.12.jar
		* CoFHAPI|modhelpers (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|tileentity (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|transport (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|world (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|audio (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui|container (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui|element (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui|element|listbox (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui|slot (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|inventory (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|render (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|render|particle (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|util (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|util|helpers (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|util|position (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|world (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|world|feature (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* DLDungeonsAPI (1.4) from DoomlikeDungeons-1.9.2-MC1.7.10.jar
		* MetallurgyAPI (2.0.2) from Metallurgy-1.7.10-4.0.9.148.jar
		* pressureAPI (1.3.0.130) from pressure-1.3.0.130-mc1.7.10.jar
		* Thaumcraft|API (4.2.2.0) from Pam's HarvestCraft 1.7.10Lb.jar
		* WailaAPI (1.2) from Waila-1.5.10_1.7.10.jar
		* ZeldaAnimationAPI (0.5) from 1.7.10-zeldaswordskills-2.5.1.jar
		* ZeldaAPI (0.4) from 1.7.10-zeldaswordskills-2.5.1.jar
		* ZeldaBlockAPI (0.3) from 1.7.10-zeldaswordskills-2.5.1.jar
		* ZeldaClientAPI (0.5) from 1.7.10-zeldaswordskills-2.5.1.jar
		* ZeldaDamageAPI (0.3) from 1.7.10-zeldaswordskills-2.5.1.jar
		* ZeldaEntityAIAPI (0.5) from 1.7.10-zeldaswordskills-2.5.1.jar
		* ZeldaEntityAPI (0.5) from 1.7.10-zeldaswordskills-2.5.1.jar
		* ZeldaGenAPI (0.3) from 1.7.10-zeldaswordskills-2.5.1.jar
		* ZeldaItemAPI (0.5) from 1.7.10-zeldaswordskills-2.5.1.jar
		* ZeldaModelAPI (0.5) from 1.7.10-zeldaswordskills-2.5.1.jar
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Player Count: 1 / 8; [EntityPlayerMP['jfrogale'/304, l='Survival (mod)', x=-1096.69, y=70.00, z=2035.27]]
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'

**Comment by coolAlias** [comment]
The Land Octorok's are from TheRedMajora's ZSS Addon mod - looks like it's outdated, so I suggest removing it if you can.

**Comment by jfrogale** [comment]
Wow, i'm an idiot. Sorry for wasting your time having to read through all that for such a simple fix. Thank you, still.

**Comment by coolAlias** [comment]
Np ;)
